examples/advanced-filter-convert-publish/README.md

```diff
@@ -60,50 +60,66 @@ Using the following setup, this example advanced **App Functions Service** can b
 
    - [ ] Run "**./simple-filter-xml**"
 
-5. Run Device Virtual service
+5. Run **Device Virtual** service
+
+   This example expects the float encoding for all random floats to be `Base64` and needs to restrict the range of values that are generated so they are easy to read. By default the device-virtual is using `Base64` for Float32 & `eNotation` for Float64 and doesn't see and range limits. Below are instructions on how to change the configuration  to meet these needs.
```

Comment:
"and doesn't see and range limits" -- wut?

"change the configuration  to meet these needs." -- remove double space